apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/logic/index.tsx

```diff
@@ -38,13 +39,25 @@ const migrateAssetsMock: PostOnboardingAction = {
     setDrawer(PostOnboardingMockAction, { id: PostOnboardingActionId.migrateAssetsMock }),
 };
 
+const customImage: PostOnboardingAction = {
+  id: PostOnboardingActionId.customImage,
+  Icon: Icons.BracketsMedium,
+  featureFlagId: "customImage",
+  title: `Personalize my ${getDeviceModel(DeviceModelId.nanoFTS).productName}`,
+  description: "By customizing the screen.",
+  actionCompletedPopupLabel: "Device personalized",
+  actionCompletedHubTitle: "That screen is looking neat.",
+  startAction: () => setDrawer(CustomImage, { isFromPostOnboardingEntryPoint: true }),
+};
+
```

Comment:
No translations?

Reply:
damn I knew I did this too fast hahaha, thanks!